_observability-plugin/notebooks.md

```diff
@@ -18,7 +18,7 @@ You can also generate [reports]({{site.url}}{{site.baseurl}}/dashboards/reportin
 
 Common use cases include creating postmortem reports, designing runbooks, building live infrastructure reports, and writing documentation.
 
-Tenants in OpenSearch Dashboards are spaces for saving notebooks and other OpenSearch Dashboards objects. For more information, see [OpenSearch Dashboards multi-tenancy]({{site.url}}{{site.baseurl}}/security-plugin/access-control/multi-tenancy/).
+Tenants in OpenSearch Dashboards are spaces for saving notebooks and other OpenSearch Dashboards objects. For more information, see [OpenSearch Dashboards multi-tenancy]({{site.url}}{{site.baseurl}}/security-plugin/multi-tenancy/multi-tenancy-config/).
```

Comment:
Since the language here directs users to general information (the "OpenSearch Dashboards multi-tenancy" page), it might be better to use this link:

({{site.url}}{{site.baseurl}}/security-plugin/security-plugin/multi-tenancy/tenant-index/)

Alternatively, you could change the language to:

For more information on configuring tenants, see [Multi-tenancy configuration]...

...and keep the link.